src/components/ActionDataSamplesCard.tsx

```diff
@@ -30,7 +38,7 @@ interface ActionDataSamplesCardProps {
   value: GestureData;
   selected: boolean;
   onSelectRow?: () => void;
-  onRecord: () => void;
+  onRecord: (recordingsToCapture: number, continuousRecording: boolean) => void;
```

Comment:
Because this is restated a lot of times as the prop is passed it might be nicer to have a RecordingOptions object.

Reply:
Agreed.

Reply:
[2b17f6f](https://github.com/microbit-foundation/ml-trainer/pull/450/commits/2b17f6fdf27353e1db17aed68c38a1f38271173d)